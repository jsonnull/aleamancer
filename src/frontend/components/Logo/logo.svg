<?xml version="1.0"?>
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 68.46 10">
<g id="Layer_2" data-name="Layer 2">
    <g id="Layer_1-2" data-name="Layer 1">
        <path d="M9.91,8.87a8.18,8.18,0,0,0,.71,1v0H7.9a1.45,1.45,0,0,0,0-.32,2.35,2.35,0,0,0-.11-.7,9.16,9.16,0,0,0-.37-1l-.21-.53H3.4l-.26.57a8.29,8.29,0,0,0-.36,1,2.45,2.45,0,0,0-.12.71,1.82,1.82,0,0,0,0,.32H0v0a6.53,6.53,0,0,0,.68-1q.31-.52.69-1.32l2.36-5.1A15.58,15.58,0,0,0,4.68.08H6.23a15.75,15.75,0,0,0,.89,2.43L9.3,7.64A10.06,10.06,0,0,0,9.91,8.87ZM4,5.92H6.66L5.4,2.83Z"/>
        <path d="M11.45,9a5.83,5.83,0,0,0,.08-1V2.64a7.19,7.19,0,0,0-.09-1.28,4,4,0,0,0-.26-.92L13.53,0a10.64,10.64,0,0,0-.22,1.4c-.05.57-.08,1.17-.08,1.8V8a5.83,5.83,0,0,0,.08,1,4.76,4.76,0,0,0,.27.93h-2.4A4.76,4.76,0,0,0,11.45,9Z"/>
        <path d="M20,6.59l-3.66.09a2.11,2.11,0,0,0,.59,1.42,1.74,1.74,0,0,0,1.21.43,2.52,2.52,0,0,0,.83-.15,5.34,5.34,0,0,0,1-.48l0,0-.5,1.74a4.06,4.06,0,0,1-1.65.34,3.24,3.24,0,0,1-2.4-.87,3.44,3.44,0,0,1-.87-2.55,4.19,4.19,0,0,1,.42-1.95A2.89,2.89,0,0,1,16,3.4,2.93,2.93,0,0,1,17.58,3a2.21,2.21,0,0,1,1.8.73,3.11,3.11,0,0,1,.62,2A6.54,6.54,0,0,1,20,6.59Zm-1.29-.87a2,2,0,0,0-.3-1.21,1,1,0,0,0-.85-.42,1,1,0,0,0-.83.39,2.28,2.28,0,0,0-.37,1.24Z"/>
        <path d="M25.15,3.54a2.1,2.1,0,0,1,.6,1.61V8a5.86,5.86,0,0,0,.08,1,4.74,4.74,0,0,0,.3.9H24V9.13A6.34,6.34,0,0,0,23,9.94h-.24a2.22,2.22,0,0,1-1.5-.46,1.54,1.54,0,0,1-.54-1.25,1.51,1.51,0,0,1,.37-1,3.91,3.91,0,0,1,1.31-.88c.51-.24,1.07-.48,1.68-.73V5.47a1.1,1.1,0,0,0-.3-.84,1.15,1.15,0,0,0-.84-.3,3,3,0,0,0-1.64.52l0,0,.5-1.57A4.57,4.57,0,0,1,23.37,3,2.49,2.49,0,0,1,25.15,3.54ZM22.65,6.9a2.05,2.05,0,0,0-.17.8,1,1,0,0,0,.3.8,1.41,1.41,0,0,0,.94.26.92.92,0,0,0,.32,0V6.9Z"/>
        <path d="M35,9a5,5,0,0,0,.08-1V5.77c0-.87-.38-1.31-1.15-1.31a1.73,1.73,0,0,0-1,.33,3.13,3.13,0,0,1,0,.35V8a5.83,5.83,0,0,0,.08,1,4.76,4.76,0,0,0,.27.93H30.93A5.57,5.57,0,0,0,31.21,9a5.87,5.87,0,0,0,.08-1V5.8a1.45,1.45,0,0,0-.28-1,1,1,0,0,0-.79-.32,1.92,1.92,0,0,0-1,.36V8a5,5,0,0,0,.08,1,4.76,4.76,0,0,0,.27.93h-2.4A6.72,6.72,0,0,0,27.4,9a5.9,5.9,0,0,0,.07-1V4.93A6.8,6.8,0,0,0,27.41,4a7.32,7.32,0,0,0-.22-.9h2.06a5,5,0,0,0-.07.89v.48l.36-.62A1.65,1.65,0,0,1,31.11,3a1.89,1.89,0,0,1,1.17.36,1.79,1.79,0,0,1,.64,1.1l.36-.62A1.66,1.66,0,0,1,34.84,3a1.91,1.91,0,0,1,1.41.54,2.17,2.17,0,0,1,.54,1.6V8a5.9,5.9,0,0,0,.07,1,6.72,6.72,0,0,0,.27.93h-2.4A4.76,4.76,0,0,0,35,9Z"/>
        <path d="M42.3,3.54a2.1,2.1,0,0,1,.6,1.61V8A5,5,0,0,0,43,9a3.65,3.65,0,0,0,.3.9H41.2V9.13a6.34,6.34,0,0,0-1.08.81h-.24a2.22,2.22,0,0,1-1.5-.46,1.54,1.54,0,0,1-.54-1.25,1.46,1.46,0,0,1,.37-1,3.91,3.91,0,0,1,1.31-.88c.5-.24,1.06-.48,1.68-.73V5.47a1.07,1.07,0,0,0-.31-.84,1.13,1.13,0,0,0-.83-.3,3,3,0,0,0-1.64.52l0,0,.51-1.57A4.53,4.53,0,0,1,40.53,3,2.48,2.48,0,0,1,42.3,3.54ZM39.81,6.9a2.05,2.05,0,0,0-.17.8,1,1,0,0,0,.29.8,1.44,1.44,0,0,0,1,.26,1,1,0,0,0,.32,0V6.9Z"/>
        <path d="M44.55,9a5.83,5.83,0,0,0,.08-1V4.93A5.49,5.49,0,0,0,44.56,4a4.89,4.89,0,0,0-.22-.9h2.07a5,5,0,0,0-.07.89v.48L47.4,3.19a2.16,2.16,0,0,1,1-.21,2,2,0,0,1,1.44.55,2.21,2.21,0,0,1,.55,1.64V8a5.83,5.83,0,0,0,.08,1,4.76,4.76,0,0,0,.27.93H48.34A4.68,4.68,0,0,0,48.61,9a5,5,0,0,0,.09-1V5.59a1.06,1.06,0,0,0-.32-.8,1.29,1.29,0,0,0-.87-.29,2.12,2.12,0,0,0-.57.08,2.66,2.66,0,0,0-.6.28V8a5.9,5.9,0,0,0,.07,1,6.72,6.72,0,0,0,.27.93h-2.4A4.76,4.76,0,0,0,44.55,9Z"/>
        <path d="M56.69,3.14,56.55,5l0,0a3.74,3.74,0,0,0-.82-.6A1.65,1.65,0,0,0,55,4.24a1.37,1.37,0,0,0-1.09.51,2.27,2.27,0,0,0-.43,1.51A2.5,2.5,0,0,0,54,8a2,2,0,0,0,1.54.65,2.93,2.93,0,0,0,.61-.06,2.66,2.66,0,0,0,.63-.2l0,0-.76,1.41A3.46,3.46,0,0,1,55,10a4,4,0,0,1-1.71-.36A2.89,2.89,0,0,1,52,8.51a3.57,3.57,0,0,1-.47-1.9A3.81,3.81,0,0,1,52,4.75a3.27,3.27,0,0,1,1.28-1.3,3.81,3.81,0,0,1,2-.47A6.44,6.44,0,0,1,56.69,3.14Z"/>
        <path d="M62.84,6.59l-3.66.09a2.15,2.15,0,0,0,.58,1.42A1.77,1.77,0,0,0,61,8.53a2.42,2.42,0,0,0,.82-.15,5.08,5.08,0,0,0,1-.48l0,0L62.3,9.66a4.06,4.06,0,0,1-1.65.34,3.24,3.24,0,0,1-2.4-.87,3.44,3.44,0,0,1-.87-2.55,4.19,4.19,0,0,1,.42-1.95A2.89,2.89,0,0,1,60.45,3a2.22,2.22,0,0,1,1.81.73,3.11,3.11,0,0,1,.62,2A6.54,6.54,0,0,1,62.84,6.59Zm-1.29-.87a2,2,0,0,0-.3-1.21,1,1,0,0,0-.86-.42,1,1,0,0,0-.82.39,2.36,2.36,0,0,0-.38,1.24Z"/>
        <path d="M64.17,9a5.83,5.83,0,0,0,.08-1V4.93A6.71,6.71,0,0,0,64.18,4a5.86,5.86,0,0,0-.22-.9H66A5,5,0,0,0,66,4v.82A4.83,4.83,0,0,1,67,3.4,1.52,1.52,0,0,1,68,3a1.42,1.42,0,0,1,.48.08l-.24,2.07a1.91,1.91,0,0,0-1-.36A1.83,1.83,0,0,0,66,5.33V8A5.83,5.83,0,0,0,66,9a4.76,4.76,0,0,0,.27.93H63.9A4.76,4.76,0,0,0,64.17,9Z"/>
    </g>
</g>
</svg>

