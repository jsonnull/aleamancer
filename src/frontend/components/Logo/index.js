// @flow
import React from 'react'
import styled from 'styled-components'
import logo from '!raw-loader!./logo.svg'

const Container = styled.div`
  svg {
    height: ${props => props.height};
    fill: currentColor;
  }
`

type Props = {
  height?: string
}
const Logo = (props: Props) => {
  const { height = '16px' } = props
  return (
    <Container height={height} dangerouslySetInnerHTML={{ __html: logo }} />
  )
}

export default Logo
