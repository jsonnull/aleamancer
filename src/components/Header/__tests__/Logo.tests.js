// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import Logo from '../Logo.js'

describe('Header Logo component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Logo />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
