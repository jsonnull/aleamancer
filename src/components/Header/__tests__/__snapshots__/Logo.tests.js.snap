// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Logo component renders correctly 1`] = `
<div
  className="sc-htpNat gYJyLH"
>
  <div
    className="sc-bwzfXH dXutcX"
    dangerouslySetInnerHTML={
      Object {
        "__html": Object {},
      }
    }
    height="12px"
  />
</div>
`;
